### PR TITLE
Fix frontend crash error

### PR DESCRIPTION
**Pull request by ajpal**
We were seeing this error:
image
If there was no previous run to compare to

Turns out, there wasn't a `main` run in the last 30 runs, so the slice of previous runs we returned didn't have a `main` run.
Fix is to make sure we have at least on `main` run in the slice.

### CODE REVIEW

File: infra/profile.py [hunk 1]
```diff
@@ -92,8 +92,11 @@ def bench(benchmark):
 def get_llvm(runMethod, benchmark):
   path = f'./tmp/bench/{benchmark}/llvm-{runMethod}/{benchmark}-{runMethod}.ll'
 
-  with open(path) as f:
-    return f.read()
+  try:
```
Comment (oflatt) [inline infra/profile.py:95]: Shouldn't we return an error here or something?

**Review by oflatt — CHANGES_REQUESTED** [review]
Good fix! Concerned about error reporting on .ll files

File: infra/profile.py (hunk 1 above)
Comment (ajpal) [reply infra/profile.py:95]: I think it's better to handle errors gracefully when possible- I wouldn't want the whole nightly run to crash if one benchmark doesn't output the llvm file. I think it's better to return `''` which we can detect on the frontend and show like "hey, this benchmark is missing the llvm"

Comment (oflatt) [reply infra/profile.py:95]: Ah yes I agree, I just want to make sure we write the code that detects the missing llvm and throws an error

**Pull request merged**